### PR TITLE
Add token configuration, XML comments

### PR DESCRIPTION
**Pull request by kylecjacob**
This PR makes it to where when the user sends a valid request (any email for the username and any string for the password until we implement mongo) to /api/login, a JWT token is returned. The token should be used in all subsequent requests in order to authenticate the user. I also added some XML documentation to the swagger doc.

### CODE REVIEW

File: MyWellnessPlannerApi/MyWellnessPlannerApi/Controllers/LoginController.cs [hunk 1]
```diff
@@ -1,16 +1,43 @@
 ﻿using Microsoft.AspNetCore.Mvc;
 using MyWellnessPlannerApi.Models;
+using MyWellnessPlannerApi.Services.Interfaces;
+using System;
```
Comment (lej656) [inline MyWellnessPlannerApi/MyWellnessPlannerApi/Controllers/LoginController.cs:4]: is this necessary

Comment (lej656) [inline MyWellnessPlannerApi/MyWellnessPlannerApi/Controllers/LoginController.cs:4]: stellar addition

Comment (kylecjacob) [reply MyWellnessPlannerApi/MyWellnessPlannerApi/Controllers/LoginController.cs:4]: ty

**Review by lej656 — APPROVED** [review]
i love iT

**Review by lej656 — APPROVED** [review]

**Pull request merged**